prime-router/src/main/kotlin/history/azure/DatabaseSubmissionsAccess.kt

```diff
@@ -40,6 +41,7 @@ class DatabaseSubmissionsAccess(private val db: DatabaseAccess = BaseEngine.data
      */
     override fun <T> fetchActions(
         organization: String,
+        orgSuffix: String?,
```

Comment:
Same as other comment, probably should be named 'orgService' to match what we call it elsewhere

Reply:
done!